src/Symfony/Component/Mailer/Bridge/Brevo/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+CHANGELOG
+=========
+
+6.3
+---
+
+* Added the bridge as a replacement of the deprecated Sendinblue one.
```

Comment:
```suggestion
* Add the bridge
```

Reply:
@PEtanguy you forgot this change when fixing requested things